README.md

```diff
@@ -53,14 +53,29 @@ cp $(go env GOPATH)/bin/ansisvg /usr/local/bin
 
 ## Fonts
 
-Note that embedded fonts might not be supported by some SVG viewers. At time of writing this is not supported by Inkscape (see https://gitlab.com/inkscape/inbox/-/issues/301).
+`ansisvg` can either use system-installed fonts (`-fontname`), link to a webfont on a HTTP server (`-fontref`) or embed a webfont from the local filesystem (`-fontfile`).
 
-## Development
+### Compatibility issues
 
-Build from cloned repo:
-```
-go build -o ansisvg main.go
-```
+* Embedded and/or linked fonts might not be supported by some SVG viewers. At time of writing this is [not supported by Inkscape](https://gitlab.com/inkscape/inbox/-/issues/301).
+
+* For SVGs that are intended to be included in websites via `<img>`, the only way to make a custom font work is [embedding it in the SVG](https://vecta.io/blog/how-to-use-fonts-in-svg).
+
+### Variations of custom fonts (regular/bold/italic)
+
+* System wide fonts (`-fontname`) get correctly rendered with variations, but when using external fonts with `-fontref` or `-fontfile` the SVG viewer knows only the regular variant and will try to render italic/bold text 'extrapolated' from it which may look different than the actual font variation. To use the actual bold/italic font variants, different woff2 files have to be used for the respective text styles which needs additional CSS code (currently not supported by `ansisvg`).
```

Comment:
Oh haven't thought about this, so one woff2 file etc can only include one variants? or it can but needs some css etc to make it work?

Reply:
Ok, maybe in the future! 😄